plugins/notifications/src/api/NotificationsClient.ts

```diff
@@ -67,6 +67,9 @@ export class NotificationsClient implements NotificationsApi {
     if (options?.createdAfter !== undefined) {
       queryString.append('created_after', options.createdAfter.toISOString());
     }
+    if (options?.minimumSeverity !== undefined) {
+      queryString.append('minimal_severity', options.minimumSeverity);
```

Comment:
We stick to camelCase wherever possible, including query parameters. Could we update this and the rest to be camelCase instead? On that topic it might be a good time to introduce an OpenAPI schema for the notifications API. See https://backstage.io/docs/openapi/01-getting-started